packages/jaeger-ui/src/components/SearchTracePage/SearchForm.test.js

```diff
@@ -392,6 +392,19 @@ describe('<SearchForm>', () => {
     btn = wrapper.find(`[data-test="${markers.SUBMIT_BTN}"]`);
     expect(btn.prop('disabled')).toBeTruthy();
   });
+
+  it('uses config.search.maxLimit', () => {
+    const maxLimit = Math.floor(Math.random() * 7000) + 3000;
+    const config = {
+      search: {
+        maxLimit,
+      },
+    };
+    window.getJaegerUiConfig = jest.fn(() => config);
+    wrapper = shallow(<SearchForm {...defaultProps} selectedService="svcA" />);
+    const field = wrapper.find(`Field[name="resultsLimit"]`);
+    expect(field.prop('props').max).toEqual(maxLimit);
```

Comment:
did you confirm that this test breaks without the fix above?

Reply:
Yes

Without this fix:

Error: expect(received).toEqual(expected)

Expected value to equal:
  4854
Received:
  {"archiveEnabled": false, "dependencies": {"dagMaxNumServices": 100, "menuEnabled": true}, "linkPatterns": [], "menu": [{"items": [{"label": "GitHub", "url": "https://github.com/uber/jaeger"}, {"label": "Docs", "url": "http://jaeger.readthedocs.io/en/latest/"}, {"label": "Twitter", "url": "https://twitter.com/JaegerTracing"}, {"label": "Discussion Group", "url": "https://groups.google.com/forum/#!forum/jaeger-tracing"}, {"label": "Gitter.im", "url": "https://gitter.im/jaegertracing/Lobby"}, {"label": "Blog", "url": "https://medium.com/jaegertracing/"}], "label": "About Jaeger"}], "search": {"maxLimit": 4854, "maxLookback": {"label": "2 Days", "value": "2d"}}, "tracking": {"gaID": null, "trackErrors": true}}